src/main/java/org/fundacionjala/coding/manuel/movies/Movie.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.manuel.movies;
+
+/**
+ * Created by Juan Manuel on 3/18/2018.
+ */
+public abstract class Movie {
+    public static final int NEW_RELEASE = 1;
+    private String title;
+
+
+    /**
+     * Constructor.
+     * @param title .
+     */
+    public Movie(String title) {
+        this.title = title;
+    }
+
+    /**
+     * Method getter title.
+     * @return title .
+     */
+    public String getTitle() {
+        return this.title;
+    }
+
+    /**
+     * Method calculate amount.
+     * @param daysRented .
+     * @return amount .
+     */
+    abstract double calculateAmount(int daysRented);
+
+    /**
+     * Method calculate frequent.
+     * @param daysRented .
+     * @return frequency .
+     */
+    public int calculateFrequentRenterPoints(int daysRented) {
+        return 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3400)